client/src/Components/Ratings-Reviews/ReviewForm.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+class ReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return(
+
+    )
```

Comment:
Will the page still run with this return being empty or does it need to be null or a dummy div?

Reply:
It is commented out in app.jsx so should be fine